### PR TITLE
tables

### PR DESCRIPTION
**Pull request by aleksdenni**
Liquibase scripts to initialize project tables

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -117,6 +117,11 @@
             <version>2.2.2</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
```
Comment (alex97po) [inline pom.xml:123]: Where liquibase version is set? I don't see properties block.

File: src/main/resources/db/liq/changelog.xml [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            <column name="passed_test_id" type="int" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="category_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="timestamp" type="int">
```
Comment (alex97po) [inline src/main/resources/db/liq/changelog.xml:116]: Why type is int?
There are dedicated types for dates in PostreSQL: 
https://www.postgresql.org/docs/current/datatype-datetime.html 
E.g., timestamp)))

File: src/main/resources/db/liq/changelog.xml [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    <changeSet  author="aleksdenni"  id="11">
+        <addForeignKeyConstraint  baseColumnNames="category_id"
+                                  baseTableName="passed_test"
+                                  constraintName="fk_passed_test_categories"
+                                  referencedColumnNames="category_id"
+                                  referencedTableName="categories"/>
+    </changeSet>
+    <!--> <-->
+    <changeSet id="12" author="aleksdenni">
+        <createTable tableName="keycloak">
```
Comment (alex97po) [inline src/main/resources/db/liq/changelog.xml:147]: Please, remove it. We don't manage Keycloak tables.

File: src/main/resources/db/liq/changelog.xml [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    <changeSet  author="aleksdenni"  id="8">
+        <addForeignKeyConstraint  baseColumnNames="question_id"
+                                  baseTableName="options"
+                                  constraintName="fk_options_questions"
+                                  referencedColumnNames="question_id"
+                                  referencedTableName="questions"/>
+    </changeSet>
+    <!--> successful? name column? <-->
+    <changeSet id="9" author="aleksdenni">
+        <createTable tableName="passed_test">
```
Comment (alex97po) [inline src/main/resources/db/liq/changelog.xml:106]: @NatMyronenko do you agree on table name? Is it descriptive enough?

File: pom.xml (hunk 1 above)
Comment (aleksdenni) [reply pom.xml:123]: I used the version from "spring-boot-dependencies-3.0.3.pom".

File: src/main/resources/db/liq/changelog.xml [hunk 5]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <!-->columns ? username <-->
+    <changeSet id="1" author="aleksdenni">
+        <createTable tableName="users">
```
Comment (NatMyronenko) [inline src/main/resources/db/liq/changelog.xml:9]: users -> change to user   (all table names & fileds should be as in the  structure (created by mentor)

Comment (NatMyronenko) [inline src/main/resources/db/liq/changelog.xml:9]: users -> change to user (all table names & fileds should be as in the structure (created by mentor)

File: src/main/resources/db/liq/changelog.xml [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="language" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <!--> <-->
+    <changeSet id="3" author="aleksdenni">
+        <createTable tableName="categories">
```
Comment (NatMyronenko) [inline src/main/resources/db/liq/changelog.xml:37]: caregories ->category

File: src/main/resources/db/liq/changelog.xml (hunk 4 above)
Comment (NatMyronenko) [reply src/main/resources/db/liq/changelog.xml:106]: table name -> (change to) user_passed_test

Comment (aleksdenni) [reply src/main/resources/db/liq/changelog.xml]: @NatMyronenko do you permit the name of the "successful" column?

Comment (NatMyronenko) [reply src/main/resources/db/liq/changelog.xml]: No, tableName "passed_test"  change to "user_passed_test", this name has a more detailed description and also corresponds to the name in the relations hierarchy created by our mentor.

Comment (NatMyronenko) [reply src/main/resources/db/liq/changelog.xml]: @aleksdenni No, tableName "passed_test" change to "user_passed_test", this name has a more detailed description and also corresponds to the name in the relations hierarchy created by our mentor.

**Pull request closed without merging**